src/sources/dataModel/taxonomic-service.json

```diff
@@ -174,20 +362,51 @@
                         "https://eosc.eu/wp-content/uploads/2022/09/New-guidelines-for-EOSC-family-branding.jpg"
                     ]
                 },
-                "schema:dateModified": {
-                    "type": "string",
-                    "description": "The date on which the Service was most recently modified or when the item's entry was modified within a DataFeed.",
-                    "format": "date-time",
-                    "examples": [
-                        "2023-07-10T12:02:00.000Z"
-                    ]
-                },
+           
                 "schema:termsOfService": {
                     "type": "string",
-                    "description": "Human-readable terms of service documentation.",
+                    "description": "URL pointing to Terms of Service, Terms of Use or Terms and Conditions, the legal agreements between a service provider and a person who wants to use that service.",
                     "examples": [
                         "http://service-name.service-provid-er.eu/TermsOfUse"
                     ]
+                },
+                "additionalProperty": {
```

Comment:
He we already have this one right?

Reply:
Oh wait this is in the service description. Might be better to put this in a separate file then and ref it in both places so you don't have duplicates

Reply:
There are already duplicates, schema:name and schema:url are also used multiple times on different levels